examples/06_scope_functions/01_let.md

```diff
@@ -0,0 +1,42 @@
+# let
+
+`let` is a Kotlin standard library function that can be used for scoping and null-checks. When called on an object, `let` executes the given functional block and returns the result of its last expression.
```

Comment:
[only a suggestion]

The Kotlin standard library function 'let' can be used for scoping and null-checks.

The Kotlin standard library includes the function 'let' which can be used for scoping and null-checks.